infra/build/Makefile

```diff
@@ -24,7 +24,7 @@ BATS_SUPPORT_VERSION := 0.3.0
 BATS_ASSERT_VERSION := 2.0.0
 BATS_MOCK_VERSION := 1.0.1
 PACKER_VERSION := 1.4.4
-TERRAGRUNT_VERSION := 0.23.6
+TERRAGRUNT_VERSION := 0.23.25
 KUSTOMIZE_VERSION := 3.6.0
 
 DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.11.4
```

Comment:
Please increment the image version tag too.

Reply:
Done